models/points.go

```diff
@@ -1038,8 +1038,8 @@ func scanFieldValue(buf []byte, i int) (int, []byte) {
 	start := i
 	quoted := false
 	for i < len(buf) {
-		// Only escape char for a field value is a double-quote
-		if buf[i] == '\\' && i+1 < len(buf) && buf[i+1] == '"' {
+		// Only escape char for a field value is a double-quote and backslash
+		if buf[i] == '\\' && i+1 < len(buf) && (buf[i+1] == '"' || buf[i+1] == '\\') {
```

Comment:
From recollection of when I did the [er-escape-backslash](https://github.com/influxdata/influxdb/compare/er-escape-backslash) branch, I don't think this quite captures the desirable behaviour. 

If the rule is "the only escapable characters are a double quote and a backslash" then what happens if we hit a backslash that isn't followed by a double quote or a backslash. For example:

```
cpu value="foo\s"
```

This current approach I think will skip over the single slash silently, but I feel like maybe it should be an error? Possibly we have discussed this before..